meta/main.yml

```diff
@@ -14,15 +14,23 @@ galaxy_info:
     - name: ArchLinux
       versions:
         - all
+    - name: Debian
+      versions:
+        - jessie
+        - stretch
     - name: EL
       versions:
         - 6
         - 7
+        - 8
     - name: Fedora
       versions:
         - 26
         - 27
         - 28
+        - 29
+        - 30
+        - 31
```

Comment:
Added Fedora 29, 30 and 31 releases.
Maybe we will have only three latest releases and can delete 26, 27, 28?